dashboard/modules/reporter/reporter_agent.py

```diff
@@ -633,6 +658,13 @@ def _generate_reseted_stats_record(self, component_name: str) -> List[Record]:
                 tags=tags,
             )
         )
+        records.append(
```

Comment:
So we will need to return the metrics that need to be reset for each component? 
What's the effect of not doing this btw? Like what is this fixed? I guess I wasn't fully aware of the problem yet.

Reply:
This is description 1 in the PR description.

Without this, the mem usage will never be cleaned up.

Reply:
it never gets reset

Reply:
so if you have an actor, ray::Actor that uses 50MB of memory, it will never get reset.